template/ci/Jenkinsfile

```
@@ -20,13 +28,26 @@ node('docker') {
         "IMAGE_TAG=${env.ghprbActualCommit.substring(0,8)}",
         "PROJECT_NAME={{Name}}-${env.BUILD_NUMBER}",
         "ROOTDIR=${global.HOST_WORKSPACE}"]) {
+        "SONAR_SCANNER_OPTS=-Djavax.net.ssl.trustStore=${HOME}/sonarcacert.ks -Djavax.net.ssl.trustStorePassword=changeit",
```

It looks like SONAR_SCANNER_OPTS aren't in the withEnv parameters.
```suggestion
        "ROOTDIR=${global.HOST_WORKSPACE}",
        "SONAR_SCANNER_OPTS=-Djavax.net.ssl.trustStore=${HOME}/sonarcacert.ks -Djavax.net.ssl.trustStorePassword=changeit",
        ]) {
```